roles/vdm/tasks/postgres/pg_config.yaml

```diff
@@ -0,0 +1,50 @@
+- name: Check for storage info ConfigMap
+  block:
+  - name: set facts 
+    set_fact:
+      dac_crunchy_storage_cm_found: false
+  - name: Look for ConfigMap
+    kubernetes.core.k8s_info:
+      kubeconfig: "{{ KUBECONFIG }}"
+      api_version: v1
+      kind: ConfigMap
+      name: dac-crunchy-storage-info
+      namespace: "{{ NAMESPACE }}"
+    register: cm_info
+  - name: set fact that ConfigMap was found
+    set_fact:
+      dac_crunchy_storage_cm_found: true
+    ignore_errors: true
+    when: "cm_info.resources"
+  - name: Use previously applied postgres storage map values
+    debug: 
+      msg:
+      - "The previous internal postgres_servers variable map values for storage class and access mode will be used."
+      - "The current internal postgres_servers variable map for storage class and access mode values will be ignored!"
```

Comment:
I know what you're trying to say here, but it's a bit muddled. You might just want to say the previously found value for the postgres server storage will be used and that the values you have in your ansible vars will be ignored because the storage class is already present in your cluster.

Reply:
Updated to say:
- "The previously found values for the postgres server storage class and access mode will be used."
- "The values you have in your ansible vars will be ignored because the storage class is already present in your cluster."